File: Mtk7688-Workshop/1.7688_Board_LED.py
#coding=utf-8
import mraa,time

x = mraa.Gpio(44)       #定義GPIO(44)為x
x.dir(mraa.DIR_OUT)

while True:
    x.write(1)          #將GPIO(44)切換至High
    time.sleep(0.2)     #延遲0.2秒
    x.write(0)          #將GPIO(44)切換至Low
    time.sleep(0.2)     #延遲0.2秒

File: node/1.7688_Board_LED.js
var mraa = require('mraa');

var x = new mraa.Gpio(44);
x.dir(mraa.DIR_OUT);

while(true){
  setTimeout(function(){
      x.write(1);
  },200);
  setTimeout(function(){
      x.write(1);
  },200);
}
